packages/eslint-plugin-gestalt/src/eslintASTFixers.js

```diff
@@ -70,21 +72,31 @@ export const renameTagFixer: RenameTagFixerType = ({
   context,
   elementNode,
   fixer,
+  gestaltImportNode,
   newComponentName,
+  replaceRegexCallback = ({ input }) => input,
```

Comment:
This seems a bit specific to `dangerouslySetInlineStyle`. I wonder if it would make more sense to build that prop (and any others) as a string outside of this function, then use RenameTagWithPropsFixerType's `additionalPropsString` arg to build it into the fixed component?

Reply:
This is the less "invasive" approach. Adding one additional props would require removing the previous style prop.  I could also reconstruct all props and alphabetize, but that would make the PR review even more opaque to the important changes. I might change the approach when I work on other Eslints, there must be a more universal way. For now, I think I wanna keep this one.

Reply:
What do you think of [the approach I took with the Box->Flex rule](https://github.com/pinterest/gestalt/pull/1628/files#diff-ec763b6499c1e98321bf995aaa9c01807756a1e9301928bdc3834af524a9b4f2)?

Avoiding regex keeps the code more readable, and it seems a lot simpler to me to keep the rule-specific logic outside of these helpers. The helpers can remove a current prop or add a new one, and the logic within the rule can determine what needs to be removed and what needs to be added (and how the new prop(s) should be formatted).

Reply:
DONE

Reply:
Nice! Looks great. This arg can now be removed, right?